service/controller/resource/prometheus/resource.go

```diff
@@ -58,6 +64,32 @@ func toPrometheus(v interface{}) (metav1.Object, error) {
 
 	name := cluster.GetName()
 	var replicas int32 = 1
+	var uid int64 = 1000
+	var gid int64 = 65534
+	var fsGroup int64 = 2000
```

Comment:
Where do those values come from? A comment would help.

Reply:
Done in the other PR